video_emotion/emotionTagger/t/test_face_emotion_extraction.py

```diff
@@ -60,18 +63,38 @@ def test_get_labels_valid_input():
                         3:'happy', 4:'sad', 5:'surprise',
                         6:'neutral'}
 
-    result = get_labels('fer2013')
+    result = get_labels()
 
     assert dataset_expected == result
 
-def test_preprocess_input():
+def test_preprocess_input(): #test if the preprocessor behaves as it should
     expected = 0.9607843137254901
 
     tf = np.float32(250) # testing float
-    result = preprocess_input(tf, True)
+    result = preprocess_input(tf)
 
     assert (expected == result)
 
-
-    
+def test_Many_Faces():
+    face_list = []
+    dirname = os.path.abspath(os.path.dirname(__file__))
+    fileName1 = os.path.join(
+                            dirname,
+                            "test_face_emotion_extraction_data/happy1.png")
+    fileName2 = os.path.join(
+                            dirname,
+                            "test_face_emotion_extraction_data/happy2.png")
+    fileName3 = os.path.join(
+                            dirname,
+                            "test_face_emotion_extraction_data/happy3.png")
```

Comment:
Please don't indent the parameters this much, when they are so long. It makes the lines very long. Do this instead:
```python
fileName3 = os.path.join(
    dirname,
    "test_face_emotion_extraction_data/happy3.png")
```